lib/timex.ex

```diff
@@ -770,27 +770,39 @@ defmodule Timex do
   Returns a boolean indicating whether the first `Timex.Comparable` occurs between the second
   and third.
 
-  By default, the `start`and `ending` bounds are *exclusive*. You can opt for inclusive bounds
-  by setting the `:inclusive` option to `true`.
+  By default, the `start`and `ending` bounds are *exclusive*. You can opt for inclusive `start`
+  and `end` bounds by setting the `:inclusive` option to `true`. You can also specify
+  `:inclusive_start` and/or `:inclusive_end` options, which will override the `:inclusive`
+  option for the given bound.
 
```

Comment:
I like the updated option! My only ask here is that the doc reflect the latest code update and consistent use of either 'end' or 'ending'.

Reply:
👍 Thanks for the reminder. I've updated the doc.